snc.sh

```diff
@@ -24,6 +24,7 @@ sudo rm -fr /var/lib/libvirt/images/crc-podman.qcow2
 CRC_INSTALL_DIR=crc-tmp-install-data
 rm -fr ${CRC_INSTALL_DIR}
 mkdir ${CRC_INSTALL_DIR}
+setfacl --default -m u:qemu:rx ${CRC_INSTALL_DIR}
```

Comment:
@anjannath it need to be recursive I think and with absolute path not relative otherwise CI is failing same way
```
sudo virt-install --name=crc-podman --vcpus=2 --ram=2048 --arch=x86_64 --import --graphics=none '--qemu-commandline=-fw_cfg name=opt/com.coreos/config,file=/home/packer/snc/crc-tmp-install-data/fcos-config.ign' --disk=size=31,backing_store=/var/lib/libvirt/images/fedora-coreos-qemu.x86_64.qcow2 --os-variant=fedora-coreos-stable --noautoconsole --quiet
ERROR    internal error: qemu unexpectedly closed the monitor: 2023-03-13T10:21:50.605332Z qemu-kvm: warning: host doesn't support requested feature: MSR(48BH).vmx-vmfunc [bit 13]
2023-03-13T10:21:50.605338Z qemu-kvm: warning: host doesn't support requested feature: MSR(48BH).vmx-rdseed-exit [bit 16]
2023-03-13T10:21:50.605343Z qemu-kvm: warning: host doesn't support requested feature: MSR(48BH).vmx-pml [bit 17]
2023-03-13T10:21:50.605349Z qemu-kvm: warning: host doesn't support requested feature: MSR(48DH).vmx-posted-intr [bit 7]
2023-03-13T10:21:50.605358Z qemu-kvm: warning: host doesn't support requested feature: MSR(48CH).vmx-ept-execonly [bit 0]
2023-03-13T10:21:50.605363Z qemu-kvm: warning: host doesn't support requested feature: MSR(48CH).vmx-eptad [bit 21]
2023-03-13T10:21:50.616372Z qemu-kvm: -fw_cfg name=opt/com.coreos/config,file=/home/packer/snc/crc-tmp-install-data/fcos-config.ign: can't load /home/packer/snc/crc-tmp-install-data/fcos-config.ign: Failed to open file “/home/packer/snc/crc-tmp-install-data/fcos-config.ign”: Permission denied 
```

Reply:
`--default` should make sure that any file created in ${CRC_INSTALL_DIR} will get the `u:qemu:rx` ACL.
```
$ mkdir snc-test
$ setfacl  -m u:qemu:rx snc-test
$ mkdir snc-test/test
$ getfacl snc-test/test
# file: snc-test/test
# owner: teuf
# group: teuf
user::rwx
group::r-x
other::r-x

$ setfacl --default  -m u:qemu:rx snc-test
$ mkdir snc-test/test2
$ getfacl snc-test/test2
# file: snc-test/test2
# owner: teuf
# group: teuf
user::rwx
user:qemu:r-x
group::r-x
mask::r-x
other::r-x
default:user::rwx
default:user:qemu:r-x
default:group::r-x
default:mask::r-x
default:other::r-x
```

Reply:
@praveenkumar did you check this in the CI and got the same error reported? I verified that on new file intended ACL is applied, same as the o/p from @cfergeau

Reply:
So using the `--default` with an absolute path also doesn't work, see https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/crc-org_snc/666/pull-ci-crc-org-snc-podman-e2e-snc/1635515315074895872

we only needed this change for the downstream CI as for the openshift CI it is working without having to apply it recursively or using the `--default` flag.

since applying it recursively works for both the openshift CI and downstream CI, i think we should go with that